docs/api-reference/layers/geojson-layer.md

```diff
@@ -366,144 +479,181 @@ In general this format is not intended to be human readable, and rather than bei
 
 At the top level the data is grouped by geometry type, into points, lines and polygons:
 
-```js
-const data = {points: {...}, lines: {...}, polygons: {...}};
+```ts
+import type {BinaryFeatureCollection} from '@loaders.gl/schema';
```

Comment:
Where is this submodule's docs? Also, the geojsonToBinary page is missing from TOS: https://loaders.gl/docs/modules/gis/api-reference/geojson-to-binary

Reply:
I don't think `@loaders.gl/schema` and `@loaders.gl/gis` has docs @ibgreen